deploy-scripts/rsk/5_2_staking_rewards_distribution.ts

```diff
@@ -21,8 +22,7 @@ async function feedStakeRewardsDistribution(hre: HardhatRuntimeEnvironment) {
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const bdx = await getBdx(hre);
   const vesting = await getVesting(hre);
-  const treasuryAddress =
-    hre.network.name.toLocaleLowerCase() === "rsk" ? constants.multisigTreasuryAddress[hre.network.name] : (await getTreasury(hre)).address;
+  const treasuryAddress = hre.network.name === "mainnetFork" ? (await getTreasury(hre)).address : constants.treasuryAddresses[hre.network.name];
```

Comment:
We do this code more than once, lets create a getTreauryAddress function in the DeployedContractsHelper

Reply:
Sababa